test/performance_test/performance.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	vegeta "github.com/tsenart/vegeta/lib"
+	"github.com/tsenart/vegeta/lib/plot"
+)
+
+// might add a with metrics bool option
+var (
+	appTestToken, workerTestToken, apiHost, proto, endpoint, reportFilePath string
+	encrypt                                                                 bool
+)
+
+func init() {
+	flag.StringVar(&appTestToken, "api_test_token", "", "access token used for api performance testing")
+	flag.StringVar(&workerTestToken, "worker_test_token", "", "access token used for worker performance testing")
+	flag.StringVar(&apiHost, "host", "localhost:3000", "host to send requests to")
+	flag.StringVar(&proto, "proto", "http", "protocol to use")
+	flag.StringVar(&endpoint, "endpoint", "ExplanationOfBenefit", "endpoint to test")
+	flag.BoolVar(&encrypt, "encrypt", true, "whether to disable encryption")
+	flag.Parse()
+
+	// create folder if doesn't exist for storing the results, maybe create env var later
+	reportFilePath = "results/"
+	if _, err := os.Stat(reportFilePath); os.IsNotExist(err) {
+		err := os.MkdirAll(reportFilePath, 0755)
```

Comment:
lol exactly it barfed, don't think 0640 will work either though. Also this might be a bigger problem throughout our app. os.ModePerm isn't caught by the linter and it's like 0777. We could probably benefit from a talk about file permissions throughout our project.

Reply:
Will `0600` work?
https://github.com/CMSgov/bcda-app/blob/master/bcdaworker/main.go#L261

If not, you can always use `nosec` (as a last resort)

Reply:
done